Failing test: Security Solution Cypress.x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations·cy·ts - AI Assistant Conversations When no conversations exist but connectors do exist, show empty convo Shows empty connector callout when a conversation that had a connector no longer does Shows empty connector callout when a conversation that had a connector no longer does

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 150000ms: Expected to find element: `[data-test-subj="connectorMissingCallout"]`, but never found it.
    at Context.eval (webpack:///./e2e/ai_assistant/conversations.cy.ts:121:40)
```

First failure: [kibana-on-merge - 9.2](https://buildkite.com/elastic/kibana-on-merge/builds/78572#0199aa2c-7145-43e9-8934-f42f82c066b9)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations·cy·ts","test.name":"AI Assistant Conversations When no conversations exist but connectors do exist, show empty convo Shows empty connector callout when a conversation that had a connector no longer does Shows empty connector callout when a conversation that had a connector no longer does","test.failCount":8}} -->

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations.cy.ts
@@ -117,10 +117,14 @@ describe('AI Assistant Conversations', { tags: ['@ess', '@serverless'] }, () => 
       cy.get(PROMPT_CONTEXT_BUTTON(0)).should('have.text', 'Alert (from summary)');
     });
     it('Shows empty connector callout when a conversation that had a connector no longer does', () => {
+      cy.intercept('PUT', '**/api/security_ai_assistant/current_user/conversations/**').as(
+        'updateConversation'
+      );
       visitGetStartedPage();
       openAssistant();
       selectConversation(mockConvo1.title);
       selectConnector(azureConnectorAPIPayload.name);
+      cy.wait('@updateConversation');
       closeAssistant();
       deleteConnectors();
       openAssistant();

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations.cy.ts
@@ -80,9 +80,9 @@ describe('AI Assistant Conversations', { tags: ['@ess', '@serverless'] }, () => 
       assertConnectorSelected('My OpenAI Connector');
     });
   });
-  // FLAKY: https://github.com/elastic/kibana/issues/237487
-  describe.skip('When no conversations exist but connectors do exist, show empty convo', () => {
+  describe('When no conversations exist but connectors do exist, show empty convo', () => {
     beforeEach(() => {
+      cy.clearLocalStorage();
       createAzureConnector();
     });
     it('When invoked on AI Assistant click', () => {

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/ai_assistant/conversations.cy.ts
@@ -5,7 +5,6 @@
  * 2.0.
  */
 
-import { RULE_MANAGEMENT_CONTEXT_DESCRIPTION } from '@kbn/security-solution-plugin/public/detection_engine/common/translations';
 import { IS_SERVERLESS } from '../../env_var_names_constants';
 import {
   assertConnectorSelected,
@@ -19,7 +18,6 @@ import {
   openAssistant,
   selectConnector,
   selectConversation,
-  selectRule,
   submitMessage,
   typeAndSendMessage,
 } from '../../tasks/assistant';
@@ -33,10 +31,9 @@ import {
 import { expandFirstAlert } from '../../tasks/alerts';
 import { ALERTS_URL } from '../../urls/navigation';
 import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
-import { visitRulesManagementTable } from '../../tasks/rules_management';
 import { deleteAlertsAndRules, deleteConnectors } from '../../tasks/api_calls/common';
 import { createRule } from '../../tasks/api_calls/rules';
-import { getExistingRule, getNewRule } from '../../objects/rule';
+import { getNewRule } from '../../objects/rule';
 import { login } from '../../tasks/login';
 import {
   CONNECTOR_MISSING_CALLOUT,
@@ -92,18 +89,6 @@ describe('AI Assistant Conversations', { tags: ['@ess', '@serverless'] }, () => 
       selectConnector(azureConnectorAPIPayload.name);
       assertConnectorSelected(azureConnectorAPIPayload.name);
       cy.get(USER_PROMPT).should('not.have.text');
-    });
-    it('When invoked from rules page', () => {
-      createRule(getExistingRule({ rule_id: 'rule1', enabled: true })).then((createdRule) => {
-        visitRulesManagementTable();
-        cy.log('NEW RULE', createdRule);
-        selectRule(createdRule?.body?.id);
-        openAssistant('rule');
-        assertNewConversation(false, `Detection Rules - Rule 1`);
-        selectConnector(azureConnectorAPIPayload.name);
-        assertConnectorSelected(azureConnectorAPIPayload.name);
-        cy.get(PROMPT_CONTEXT_BUTTON(0)).should('have.text', RULE_MANAGEMENT_CONTEXT_DESCRIPTION);
-      });
     });
     it('When invoked from alert details', () => {
       createRule(getNewRule());

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/screens/ai_assistant.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/screens/ai_assistant.ts
@@ -10,7 +10,7 @@ export const ADD_QUICK_PROMPT = '[data-test-subj="addQuickPrompt"]';
 export const AI_ASSISTANT_BUTTON = '[data-test-subj="assistantNavLink"]';
 export const ASSISTANT_CHAT_BODY = '[data-test-subj="assistantChat"]';
 export const CHAT_CONTEXT_MENU = '[data-test-subj="chat-context-menu"]';
-export const CHAT_ICON = '[data-test-subj="newChat"]';
+export const CHAT_ICON = '[data-test-subj="plusCircle"]';
 export const CHAT_ICON_SM = '[data-test-subj="newChatByTitle"]';
 export const CLEAR_CHAT = '[data-test-subj="clear-chat"]';
 export const CLEAR_SYSTEM_PROMPT = '[data-test-subj="clearSystemPrompt"]';

